common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -10,10 +10,27 @@
     ))">
 
         @defining(if(!host.isEmpty) "&origin=" + host else "") { origin =>
-            <iframe id="youtube-@media.assets.head.id" width="100%" height="100%"
+            <iframe class="atom--media--youtube--iframe" id="youtube-@media.assets.head.id" width="100%" height="100%"
             src="https://www.youtube.com/embed/@media.assets.head.id?modestbranding=1&enablejsapi=1&rel=0&showinfo=0@origin" frameborder="0"
             allowfullscreen="">
             </iframe>
+
+            @media.posterUrl.map { image =>
+                <div class="atom--media--youtube--overlay">
```

Comment:
@NataliaLKB I followed the pattern used in the existing classname 'atom--media--youtube' which was already in use in this template, however I agree the classnames if we're following BEM are misleading and I'm happy to correct this. So if 'atom--media--youtube' is an element in it's own right then I'll rename it to 'youtube-atom' and all sub elements to use '__' eg. 'youtube-atom__overlay' as suggested!